raster-foundry/raster-foundry
# Remove shapefile support, CMR

## Overview
Remove shapefile imports / exports (apis, frontend)
Remove CMR from frontend

### Checklist


## Testing Instructions

- run `./sbt api/assembly`
- Start the server
- Verify that all shapefile endpoints are missing
- Verify that adjacent (list etc) endpoints function as intended
- Verify that no shapefile upload UI components / fragments remain on the frontend

Closes https://github.com/azavea/raster-foundry-platform/issues/879
